app/modules/individuals/resources.py

```diff
@@ -217,11 +217,58 @@ def patch(self, args, individual):
         context = api.commit_or_abort(
             db.session, default_error_message='Failed to update Individual details.'
         )
+
         with context:
             parameters.PatchIndividualDetailsParameters.perform_patch(
                 args, obj=individual
             )
-            db.session.merge(individual)
+
+        # copycat the pattern established in sightings module
+        edm_count = 0
+        for arg in args:
+            if (
+                'path' in arg
+                and arg['path'][1:]
```

Comment:
change above means that this nasty slice removing the / should be dispensed with (just remove the [1:])